### DISCUSSION TITLE
System error: SyntaxError: Invalid regular expression: /?/: Nothing to repeat

### DISCUSSION THREAD

**Discussion by Jikexiaobai** [AlistGo/alist#2399] [Q&A] [answered]
访问会报错 System error: SyntaxError: Invalid regular expression: /?/: Nothing to repeat

强制刷新几下可以正常访问，随便点进一个目录下，又会报错这个

![image](https://user-images.githubusercontent.com/49304819/202418137-9ec31c8a-b502-46f3-82bc-8789b1a8ae93.png)

**Comment by xuanyu1937** [comment]
解决了吗？我的也是这个问题

**Reply by Jikexiaobai** [reply]
并没有

**Reply by Jikexiaobai** [reply]
已解决

**Reply by wszhxz** [reply]
今天也莫名其妙的报错“ System error: SyntaxError: Invalid regular expression: /?/: Nothing to repeat”，确实是油猴插件中一个自动答题脚本的问题，将域名添加到排除列表里后，alist就正常了

**Comment by Mosazh** [comment]
我也遇到了这个问题，在edge上完全访问不了，火狐就又可以。

**Reply by xuanyu1937** [reply]
在edge上可以访问后台
![image](https://user-images.githubusercontent.com/117364132/203676888-e1b71dde-db8c-47f3-a24a-f28418c508b1.png)

**Comment by blankjun02** [comment]
找到问题了，我是把油猴关掉之后好了，似乎是油猴脚本引起的

**Reply by Jikexiaobai** [reply]
谢谢，找到了问题

**Reply by haotzops** [reply]
感谢，真的牛，这都能查到

**Comment by Jikexiaobai** [comment, marked as answer]
从楼上的回复，我尝试了一下 确实是和油猴类插件有关，本人是因为下图中 脚本猫中的一个答题脚本导致的，应该是脚本问题和油猴类插件没关系（个人猜测）

![image](https://user-images.githubusercontent.com/49304819/204142855-6cc56137-4227-4704-9965-e5ffcde45ede.png)
![image](https://user-images.githubusercontent.com/49304819/204143116-515513f0-ba2f-46a6-9e97-69f087438d0c.png)

**Reply by blankjun02** [reply]
> 从楼上的回复，我尝试了一下 确实是和油猴类插件有关，本人是因为下图中 脚本猫中的一个答题脚本导致的，应该是脚本问题和油猴类插件没关系（个人猜测）
> 
> ![image](https://user-images.githubusercontent.com/49304819/204142855-6cc56137-4227-4704-9965-e5ffcde45ede.png) ![image](https://user-images.githubusercontent.com/49304819/204143116-515513f0-ba2f-46a6-9e97-69f087438d0c.png)

对的，我的表述不当，我也是在停用了某答题脚本后解决了这个问题。

**Comment by 1435646097** [comment]
对，亲测是和油猴的答题脚本冲突了，关了油猴的脚本就可以了
